src/jdk.jshell/share/classes/jdk/internal/jshell/tool/resources/l10n_de.properties

```diff
@@ -179,7 +180,7 @@ jshell.console.empty = \nLeerer Eintrag. Ein einzelner gültiger Ausdruck oder e
 im\n                            Modulpfad an, wenn <module> ALL-MODULE-PATHs lautet\n    --enable-native-access\n                          Ermöglicht Ausführung eingeschränkter nativer Methoden durch Code\n    --enable-preview      Code kann Vorschaufeatures in diesem Release nutzen\n    --startup <Datei>      Ersetzung der Startdefinitionen mit einer Ausführung\n    --no-startup          Startdefinitionen werden nicht ausgeführt\n    --feedback <Modus>     Gibt den anfänglichen Feedbackmodus an. Der Modus kann\n                            vordefiniert (Silent, Concise, Normal oder Verbose) oder\n                            vorab benutzerdefiniert sein\n    -q                    Stilles Feedback. Identisch mit: --feedback concise\n    -s                    Äußerst stilles Feedback. Identisch mit: --feedback silent\n    -v                    Verbose-Feedback. Identisch mit: --feedback verbose\n    -J<flag>              Übergibt <flag> an das Laufzeitsystem, hat aber keine Auswirkungen\n                            auf die Ausführung von Code-Snippets. Um Kennzeichen anzugeben,\n                            die die Ausführung von Code-Snippets beeinflussen, verwenden Sie\n                            -R<flag>. Verwenden Sie alternativ dazu -J<flag> mit\n                            --execution local.\n    -R<flag>              Übergibt <flag> nur dann an das Laufzeitsystem, wenn\n                            Code-Snippets ausgeführt werden. Beispiel: -R-Dfoo=bar\n                            bedeutet, dass die Ausführung des Snippets\n                            System.getProperty("foo") "bar" zurückgibt.\n    -C<flag>              Übergibt <flag> an den Java-Compiler in JShell.\n                            Beispiel: -C-Xlint aktiviert alle empfohlenen\n                            LINT-Warnungen, und -C--release=<N> kompiliert für\n                            Java SE N, wie wenn --release N angegeben wird.\n                            Verwenden Sie ein -C pro Compiler-Kennzeichen oder Kennzeichenargument\n    --version             Gibt Versionsinformationen aus und beendet den Vorgang\n    --show-version        Gibt Versionsinformationen aus und setzt den Vorgang fort\n    --help, -?, -h        Gibt diese Zusammenfassung der Standardoptionen aus und beendet den Vorgang\n    --help-extra, -X      Gibt Hilfetext zu Nicht-Standardoptionen aus und beendet den Vorgang\n\nEin Dateiargument kann ein Dateiname oder einer der vordefinierten Dateinamen sein: DEFAULT,\nPRINTING, TOOLING oder JAVASE.\nEine Ladedatei kann auch "-" zur Angabe einer Standardeingabe ohne interaktiven I/O sein.\n\nWeitere Informationen zu den Auswertungskontextoptionen (--class-path,\n--module-path und --add-modules) finden Sie unter:\n\t/help context\n\nEin Pfad listet die zu durchsuchenden Verzeichnisse und Archive auf. Verwenden Sie unter Windows ein\nSemikolon (;), um Elemente im Pfad zu trennen. Verwenden Sie auf anderen Plattformen einen\nDoppelpunkt (:), um Elemente zu trennen.\n
```

Comment:
> Any particular reason you know that the tool had the definitions adjusted? I see some of the changes look like like command line options not being translated, but it's a bit hard to tell. Do you know if the reverted words should be translated (and possibly will be re-translated in future drops?) or should they be left untranslated?

@alisenchung While these command line options are untranslated, which initially may seem confusing, it appears to be the translation tool standardizing the definitions with the Japanese and Chinese localized definitions. We should leave them untranslated.

Reply:
This is what I've determined as well. I planned to leave these untranslated to reduce noise between drops as well.

Reply:
While we do not translate Java keywords (when used as such, e.g. "`class` not allowed here"), and option names (e.g. `--class-path`) we have typically translated other text that is not mandated by the language or tools, (e.g. `the name of this class`, `<args>`, `<class-path>`.

This change seems to be going backwards.

Whatever the policy we use for translating (or not translating) parts of a message, we should be consistent across all tools and documents.

Reply:
The go translation team dictates which of these segments within a file actually get updated. I can manually revert the segments if preferred, but these changes will continue appearing for future drops with the current system. 

I can potentially try to communicate to the other team to differentiate between keywords & option names versus some of these other words, but let me know how you'd like me to proceed with this.

Reply:
@jonathan-gibbons I just realized that this change to the German file matches the file with the Japanese and Chinese localized files. By that I mean the "backwards" translations are also present for the Japanese and Chinese files.

However, the previous versions of the Japanese and Chinese files already have these bits translated. So in summary, I can revert the German file back to German but the Japanese and Chinese files will still be English for these parts because I wouldn't know what to revert them to in their local language.

As a result, I actually think it might be better to change this German file as shown in the current version of the PR just to remain consistent with the other localized versions of this file. We can still bring this up to the go team as an issue, but I think this is the best solution now for this drop at least.

Reply:
No one else except people reading this PR will spot inconsistencies between translations in different languages. So inconsistencies across languages will not be obvious to end users.

But anyone (end-user) using the tool in any one of these languages is likely to spot inconsistencies in the translations for that one language.

Reply:
@jonathan-gibbons I have gone back and manually undid some of the English translations back to its former German word. Most occurrences of this are `<flag>` to `<Kennzeichen>`, but some others were changed back as well. Could you re-review this?